tests/test_389ds.py

```diff
@@ -9,7 +9,7 @@
 def test_ldapwhoami(auto_container_per_test):
     auto_container_per_test.connection.run_expect(
         [0],
-        "dsconf localhost backend create --suffix dc=example,dc=com --be-name userRoot",
+        "sleep 10 && dsconf localhost backend create --suffix dc=example,dc=com --be-name userRoot",
```

Comment:
because pytest needs to wait until the container has finished starting up. we used to use HEALTHCHECK for this, but this no longer exists in OCI formats.